runners/portability/java/build.gradle

```diff
@@ -50,27 +50,32 @@ dependencies {
   validatesRunner project(path: project.path, configuration: "testRuntime")
 }
 
-
 project.evaluationDependsOn(":sdks:java:core")
+project.evaluationDependsOn(":sdks:python")
 project.evaluationDependsOn(":runners:core-java")
 
-ext.localJobServicePidFile = "${project.buildDir}/local_job_service_pid"
+ext.pythonSdkDir = project.findProject(":sdks:python").getProjectDir().getAbsoluteFile().toString()
+
+// If this is set via -P then we assume it is already running and will not start it
+def needsLocalJobService = !project.hasProperty("localJobServicePortFile")
```

Comment:
Noting that my previous approach did not work. The property was always set by the time the conditional was checked. I had only verified it against a job service that I launched externally so did not catch that the logic was flawed. This approach does work, by setting `needsLocalJobService` earlier.